packages/notte-core/src/notte_core/actions/__init__.py

```
@@ -0,0 +1,149 @@
+"""Actions module - provides action classes and TypedDicts for type-safe action execution."""
+
+# Import all action classes from actions.py
+from notte_core.actions.actions import (  # noqa: F401
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)